examples/lbfgs/driver.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import scipy.optimize
```

Comment:
Should we install scipy in the setup script? Perhaps not... we probably won't want to install all of the libraries that we use in example applications (e.g., TensorFlow).